github/repos_contents.go

```diff
@@ -72,6 +74,28 @@ func (r *RepositoryContent) Decode() ([]byte, error) {
 	return o, nil
 }
 
+// GetContent returns the content of r, decoding it if necessary.
+func (r *RepositoryContent) GetContent() (string, error) {
```

Comment:
What do you think about gradually switching over to Go's naming policies as new APIs are added?
So for example, this would be `Content` instead of `GetContent`?
https://golang.org/doc/effective_go.html#Getters
If you want to stay consistent with the rest of `go-github`, that is fine with me too.
"I saw an opportunity and I seized it." :smiley: